mowgli_etl/pipeline/wdc/wdc_product_dimensions.py

```diff
@@ -0,0 +1,21 @@
+from typing import NamedTuple, Optional
+
+
+class WdcProductDimensions(NamedTuple):
+    """
+    Should we handle volume, mass dimensions?
+    How do we handle multiple dimensions?
+    How do we handle clothing dimensions?
+    """
+    depth: Optional[float]
```

Comment:
Since you have a _unit field for every dimension, just create another NamedTuple, WdcProductDimension, which is
```
class WdcProductDimension(NamedTuple):
    value: float # must be set if this model is set
    unit: Optional[str]
```

Then in this model, you can do `depth: Optional[WdcProductDimension]`.

Reply:
Also, use `Decimal` instead of `float`.